source/ed/connectors/fusio_parser.cpp

```diff
@@ -325,8 +325,8 @@ std::vector<ed::types::VehicleJourney*> TripsFusioHandler::get_split_vj(Data& da
     }
     ed::types::Route* route = it->second;
 
-    auto vp_it = gtfs_data.tz.vp_by_name.lower_bound(row[service_c]);
-    if(vp_it == gtfs_data.tz.vp_by_name.end()) {
+    auto vp_range = gtfs_data.tz.vp_by_name.equal_range(row[service_c]);
+    if(vp_range.first == gtfs_data.tz.vp_by_name.end() || vp_range.first->first != row[service_c]) {
```

Comment:
no, you want to know if the range is empty, ie `vp_range.first == vp_range.second`.

better to add a function like that in gtfs_parser.h:
```c++
template<typename T> bool
empty(const std::pair<T, T>& r) {
    return r.first == r.second;
}
```
and then use it.